OptiTypePipeline.py

```diff
@@ -139,10 +139,15 @@ def get_types(allele_id):
             return table.loc[aa[0]]['4digit']
         else:
             return table.loc[aa[0]]['4digit']  #+ '/' + table.loc[aa[1]]['4digit']
-            
-            
-            
-            
+
+def get_num_threads(configured_threads):
+  try:
+    import psutil
+  except ImportError:
```

Comment:
Here you have to check for `NotImplementedError` if you changed to multiprocessing